audit/projects/k8s-staging-sched-simulator/description.json

```diff
@@ -0,0 +1,11 @@
+{
+  "createTime": "2021-09-09T13:31:52.066Z",
+  "lifecycleState": "ACTIVE",
+  "name": "k8s-staging-sched-simulator",
+  "parent": {
+    "id": "758905017065",
+    "type": "organization"
+  },
+  "projectId": "k8s-staging-sched-simulator",
+  "projectNumber": "575095402180"
+}
```

Comment:
Expected, this was https://github.com/kubernetes/k8s.io/pull/2681 getting deployed